### PR TITLE
Accepting an invitation should return invitation details

### PR DESCRIPTION
**Pull request by alekshs**

### CODE REVIEW

File: openapi-v1.yaml [hunk 1]
```diff
@@ -2854,6 +2854,58 @@ definitions:
...
+            additionalProperties:
+              type: string
+      organization:
+        description: the organization that the user was invited to
+        x-omitempty: true
+        properties:
+          id:
+            type: string
+            description: organization id
+          namespace:
```
Comment (anastasop) [inline openapi-v1.yaml:2892]: nit: Add `Name` It is of course the same as namespace but with it all payloads will have similar attributes

File: openapi-v1.yaml [hunk 2]
```diff
@@ -2854,6 +2854,58 @@ definitions:
...
+            description: the namespace that the array belongs to
+          name:
+            type: string
+            description: name of the array
+          file_type:
+            $ref: "#/definitions/FileType"
+            description: array file type
+            x-omitempty: true
+            x-nullable: true
+          file_properties:
```
Comment (anastasop) [inline openapi-v1.yaml:2879]: why we need file properties?

File: openapi-v1.yaml [hunk 3]
```diff
@@ -7041,8 +7093,10 @@ paths:
       description: Accepts invitation
       operationId: acceptInvitation
       responses:
-        204:
+        200:
```
Comment (anastasop) [inline openapi-v1.yaml:7096]: This breaks compatibility. Probably nobody will notice but consider adding a new route

File: openapi-v1.yaml (hunk 2 above)
Comment (anastasop) [inline openapi-v1.yaml:2879]: why we need file_properties for the redirect?

File: openapi-v1.yaml [hunk 4]
```diff
@@ -2854,6 +2854,58 @@ definitions:
...
+          id:
+            type: string
+            description: array id
+          namespace:
+            type: string
+            description: the namespace that the array belongs to
+          name:
+            type: string
+            description: name of the array
+          file_type:
```
Comment (anastasop) [inline openapi-v1.yaml:2874]: `file_type` is internal. Maybe a better name like `asset_name` or `asset_type` or whatever

**Review by anastasop — CHANGES_REQUESTED** [review]

File: openapi-v1.yaml (hunk 4 above)
Comment (alekshs) [reply openapi-v1.yaml:2874]: file_type is a property of ArrayInfo definition

File: openapi-v1.yaml (hunk 2 above)
Comment (anastasop) [reply openapi-v1.yaml:2879]: I don't like exposing the full properties just for this. Propose alternatives if they don't cause implementation difficulties

Comment (alekshs) [reply openapi-v1.yaml:2879]: What if we limit the `file_properties` to only contain the `is_dashboard` indicator?

**Comment by alekshs** [comment]
This should be an internal operation. Closing.

**Pull request closed without merging**